plugins/github-issues/src/components/GitHubIssues/GitHubIssues.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+
+import { Box, IconButton, Typography } from '@material-ui/core';
+import { InfoCard, Progress } from '@backstage/core-components';
+import RefreshIcon from '@material-ui/icons/Refresh';
+
+import { useEntityGitHubRepositories } from '../../hooks/useEntityGitHubRepositories';
+import {
+  RepoIssues,
+  useGetIssuesByRepoFromGitHub,
+} from '../../hooks/useGetIssuesByRepoFromGitHub';
+
+import { IssueList } from './IssuesList';
+import { NoRepositoriesInfo } from './NoRepositoriesInfo';
+
+export type PluginMode = 'page' | 'card';
+
+export type Props = {
+  mode: PluginMode;
+  itemsPerPage?: number;
+  itemsPerRepo?: number;
+};
+
+export const GitHubIssues = ({
+  itemsPerPage = 10,
```

Comment:
Prefer destructuring inside the component rather than here, as it makes the `api-report.md` cleaner.

Reply:
No problem. However, I'm passing default props here. Can you please point me to an example with a preferable way of doing this?

Reply:
```ts
export const GitHubIssues = (props: GitHubIssuesProps) => 
 const { itemsPerPage = 10, itemsPerRepo = 40 } = props;
}
```

Should do it :)

Reply:
done [ffd5e47](https://github.com/backstage/backstage/pull/12875/commits/ffd5e47fb5c6338aa0e6965d687d0028e0baabb5)